pkg/stats/monitor.go

```diff
@@ -203,8 +211,30 @@ func (m *Monitor) updateEgressStats(idle float64, usage map[int]float64) {
 
 		if procStats.egressID != "" {
 			m.promProcCPULoad.With(prometheus.Labels{"egress_id": procStats.egressID}).Set(cpuUsage)
+			if cpuUsage > maxUsage {
```

Comment:
if the maxUsage instance has not exceeded quota limits that are defined, it seems that we shouldn't kill it?